src/hawkbit-client.c

```diff
@@ -555,8 +556,8 @@ static gpointer download_thread(gpointer data)
         }
 
         // notify hawkbit that download is complete
-        msg = g_strdup_printf("Download complete. %.2f MB/s",
-                              (artifact->size / ((double)(end_time - start_time) / 1000000)) / (1024 * 1024));
+        msg = g_strdup_printf("Download complete. %" CURL_FORMAT_CURL_OFF_T " MB/s",
```

Comment:
This will result in having only integer numbers for speed in MB/s which is very likely to become simply '0' for slower connections or small payloads. I would like to leave the prior %.2f format with the double cast as it was before.

Reply:
Agreed, I'll use the prior format.

Reply:
Done.